src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/SnsService.kt

```diff
@@ -61,7 +61,7 @@ class SnsService(hmppsQueueService: HmppsQueueService, private val objectMapper:
 }
 
 data class AdditionalInformation(
-  val adjudicationNumber: String? = null,
+  val chargeNumber: String? = null,
```

Comment:
probably best to rename this now? in line with the rest of system as adjudication number will be removed completely in coming weeks